### PR TITLE
Added a script to pull Plex items in multiple ways

### PR DESCRIPTION
**Pull request by jvonhoff**
Namely, I was missing the ability to add items from Collections, which is one mechanism I use to group my shows/movies.  With this, I can create a channel using the "70s & 80s TV" collection, as well as optionally adding "Paramount Studios" and even "Westerns" to the mix, if I wanted to get really specific.

Let me know if you have issues or thoughts, and thanks for laying all the groundwork!

### CODE REVIEW

**Comment by nwithan8** [comment]
Looks great on first pass! I'll test it out later and merge it in. Thanks for the contribution!

**Comment by morlanth** [comment]
Would it be possible to support playlists with this too? The shuffle feature is what I'm primarily using this for.

File: scripts/channel_builder.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+                    print(f"{genre} - {item.title}")
+                all_items.extend(genre_items)
+
+    if args.collection is not None:
+        for collection in args.collection:
+            found_coll = plex_server.library.section(section).search(title=collection, libtype='collection')
+            if found_coll and len(found_coll) == 1:
+                print("Matching COLLECTION items:")
+                for item in found_coll[0].children:
+                    print(f"{collection_parm} - {item.title}")
```
Comment (nwithan8) [inline scripts/channel_builder.py:90]: What is ``collection_parm``? Should this be ``collection``?

**Comment by nwithan8** [comment]
@jvonhoff Sorry I took a while to circle back to this. Check out my one comment. Other than that, looks and works great! I'll merge as soon as that comment is resolved.

**Comment by nwithan8** [comment]
> Would it be possible to support playlists with this too? The shuffle feature is what I'm primarily using this for.

Are you looking to just copy a playlist to dizqueTV and shuffle it?

**Comment by morlanth** [comment]
> 
> 
> > Would it be possible to support playlists with this too? The shuffle feature is what I'm primarily using this for.
> 
> Are you looking to just copy a playlist to dizqueTV and shuffle it?

Yes

**Comment by nwithan8** [comment]
> > 
> > 
> > > Would it be possible to support playlists with this too? The shuffle feature is what I'm primarily using this for.
> > 
> > Are you looking to just copy a playlist to dizqueTV and shuffle it?
> 
> Yes

I can add a shuffle feature to the ``playlist_to_channel.py`` script real quick.

**Comment by nwithan8** [comment]
> > > > Would it be possible to support playlists with this too? The shuffle feature is what I'm primarily using this for.
> > > 
> > > 
> > > Are you looking to just copy a playlist to dizqueTV and shuffle it?
> > 
> > 
> > Yes
> 
> I can add a shuffle feature to the `playlist_to_channel.py` script real quick.
 
Done: https://github.com/nwithan8/dizqueTV-scripts/blob/master/scripts/playlist_to_channel.py

File: scripts/channel_builder.py (hunk 1 above)
Comment (jvonhoff) [reply scripts/channel_builder.py:90]: Yes, you're right.  I think I changed the name trying to avoid confusion, but missed one.

**Comment by jvonhoff** [comment]
I fixed the collection variable name, and also added the option to have the plex token as a parameter, rather than having to update the script.

**Comment by jvonhoff** [comment]
Any issue with the PR?  I have another change to the new channel_builder.py script (--force so that it isn't interactive, for scripting purposes).  Also considering adding a search by Title (--section "TV Shows" --title "Friends" "Seinfeld" "The Office (US)") to build a channel from single/multiple shows/movies that aren't collected by Genre, Collection, etc.

**Comment by nwithan8** [comment]
Sorry, forgot about this PR. Merging now.

**Pull request merged**